### PR TITLE
Update package version on automatic API revision

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Update package version on API revision even if API surface matches to reflect correct package version info. This change along with tagging a revision after package release will ensure that package version is updated correctly(A separate change from updating package version).

### CODE REVIEW

**Review by chidozieononiwu — APPROVED** [review]

File: src/dotnet/APIView/APIViewWeb/Managers/APIRevisionsManager.cs
Comment (maririos) [inline src/dotnet/APIView/APIViewWeb/Managers/APIRevisionsManager.cs:834]: just making sure, does `revision.Files[0]` always exists? or would it be better to check and avoid a null pointer

Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/Managers/APIRevisionsManager.cs:834]: It should always exists for a revision. Otherwise a revision is does not have anything to show in APIView.

**Comment by praveenkuttappan** [comment]
@chidozieononiwu @maririos I have pushed additional changes to update label and also to add a new package version param to send actual package version instead of daily build alpha version from CI. I have fixed another issue in copying approval status to automatic review.

File: src/dotnet/APIView/APIViewWeb/Managers/APIRevisionsManager.cs
Comment (maririos) [reply src/dotnet/APIView/APIViewWeb/Managers/APIRevisionsManager.cs:834]: One eye is twitching bc this is a function that anyone can call and I don't want people to get cunexpected behavior. But I understand that it doesn't make sense and it should be good

File: src/dotnet/APIView/APIViewWeb/Controllers/AutoReviewController.cs
Comment (maririos) [inline src/dotnet/APIView/APIViewWeb/Controllers/AutoReviewController.cs:38]: would it be helpful to document in which cases the function expects `packageVersion` not to be `null`?

**Review by maririos — APPROVED** [review]

File: src/dotnet/APIView/APIViewWeb/Controllers/AutoReviewController.cs
Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/Controllers/AutoReviewController.cs:38]: I added it as null for now until all individual caller from language repo starts to send it. Once change is merged to individual repos, I will change this as required param.

**Pull request merged**
